app/db/queries.py

```diff
@@ -5,44 +5,71 @@
 
 
 def get_messages_that_contain_word(word: str) -> List[Message]:
-    pass
+    cont_result = list(Message.objects.filter(text__icontains=word))
```

Comment:
These variables make code more complicated for me. Also, their names are not descriptive enough. 
For example:
`cont_result` - what is cont
`start_result` - the start of which result
`message_date` - it's seems that date should be stored in such variable not a list of users
...
So, I think it's better not to use extra variables